applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/volume/bt_vol_ctlr.c

```diff
@@ -177,13 +321,18 @@ int bt_vol_ctlr_down(void)
 	int ret;
 
 	for (int i = 0; i < ARRAY_SIZE(vcs_client_peer); i++) {
-		if (vcs_client_peer[i] != NULL) {
-			ret = bt_vcp_vol_ctlr_unmute_vol_down(vcs_client_peer[i]);
-			if (ret) {
-				LOG_WRN("Failed to volume down for remote channel %d, ret "
-					"= %d",
-					i, ret);
-			}
+
+		if (!vcs_peer_alive(vcs_client_peer[i].peer)) {
+			continue;
+		}
+
+		int vol = vcs_client_peer[i].master_vol - CONFIG_BT_AUDIO_VOL_STEP_SIZE;
+
+		vol = CLAMP(vol, 0, __UINT8_MAX__);
```

Comment:
```suggestion
		vol = CLAMP(vol, 0, UINT8_MAX);
```

Minor comment, but you should use the non-underscore versions